### ISSUE TITLE
mac app is too eager to spawn new window

### ISSUE THREAD

**Issue by jvasileff** [gitterHQ/gitter#640]
Mac apps without active windows generally do nothing when selected with ⌘-Tab, but Gitter:

* creates a new window is created if none exists, or
* un-minimizes the minimized Gitter window

I understand *why* this might have been done - it is sometimes confusing to activate, for instance, Mail.app and have nothing happen until you take further action like click on the Dock icon. But, that behavior is at least standard.

The problem with Gitter's approach is that at times the Gitter window "just pops up" unexpectedly when Gitter is activated implicitly due to another app being closed. So I think it would be better to remove this non-standard behavior. If there is a usability problem with the standard approach, it is OS X's problem, not Gitter's.

**Comment by jnehlmeier** [comment]
Yeah its really a bit annoying when that Gitter window pops up when you do not expect it. If you are unlucky this can happen quite a lot.

Please stick to standard OS X behavior.
